packages/ai/tests/chat-mcp-manager.test.ts

```
@@ -3,7 +3,12 @@ import {
   MCPDuplicateToolNameError,
   MCPManager,
 } from '../src/activities/chat/mcp/manager'
+import {
+  executeServerTool,
+  type ToolExecutionMiddlewareHooks,
```

ESLint

[error] Member 'ToolExecutionMiddlewareHooks' of the import declaration should be sorted alphabetically.

(sort-imports)